CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Changed
 
 - Updated cluster-autoscaler to upstream version [1.16.2](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.16.2).
+- Updated deployment to support Azure autodiscover.
```

Comment:
@fiunchinho I think this should be under an Added heading. Can we go with this text?

> Added support for Azure with auto-discovery.

Reply:
Please, take a look now.